src/nwb_datajoint/common/common_sortingview.py

```diff
@@ -0,0 +1,155 @@
+        
+        return sortingview_sorting_id
+    
+    def add_metrics_to_sorting(self, key: dict, metrics: dict,
+                               sortingview_sorting_id: str=None):
+        """Adds a metrics to the specified sorting.
+        
+        Parameters
+        ----------
+        key : dict
+        metrics : dict
+            Quality metrics.
+            Key: name of quality metric
+            Value: another dict in which key: unit ID (must be str),
+                                         value: metric value (float)
+        sortingview_sorting_id : str, optional
+            if not specified, just uses the first sorting ID of the workspace
+        """
+        
+        # check that the unit ids are str
+        for metric_name in metrics:
+            unit_ids = metrics[metric_name].keys()
+            assert np.all([isinstance(unit_id, int)] for unit_id in unit_ids)
+        
+        # the metrics must be in this form to be added to sortingview 
+        external_metrics = [{'name': metric_name,
+                             'label': metric_name,
+                             'tooltip': metric_name,
+                             'data': metric} for metric_name, metric in metrics.items()]    
+        
+        workspace_uri = (self & key).fetch1('workspace_uri') 
+        workspace = sv.load_workspace(workspace_uri)
+        if sortingview_sorting_id is None:
+            print('sortingview sorting ID not specified, using the first sorting in the workspace...')
+            sortingview_sorting_id = workspace.sorting_ids[0]
+            
+        workspace.set_unit_metrics_for_sorting(sorting_id=sortingview_sorting_id, 
+                                               metrics=external_metrics)  
+    
+    def set_snippet_len(self, key: dict, snippet_len: int):
+        """Sets the snippet length of a workspace specified by the key
+        
+        Parameters
+        ----------
+        key : dict
+            defines a workspace
+        """
+        workspace_uri = (self & key).fetch1('workspace_uri')
+        workspace = sv.load_workspace(workspace_uri) 
+        workspace.set_snippet_len(snippet_len)
+
+    def url(self, key, sortingview_sorting_id: str=None):
+        """Generate a URL for visualizing the sorting on the browser
+        
+        Parameters
+        ----------
+        key : dict
+            defines a workspace
+        sortingview_sorting_id : str, optional
+            sortingview sorting ID to visualize; if None then chooses the first one
+            
+        Returns
+        -------
+        url : str
+        """
+
+        workspace_uri = (self & key).fetch1('workspace_uri')
+        workspace = sv.load_workspace(workspace_uri)
+        recording_id = workspace.recording_ids[0]
+        if sortingview_sorting_id is None:
+            sortingview_sorting_id = workspace.sorting_ids[0]
+        url = workspace.spikesortingview(recording_id=recording_id, 
```

Comment:
Very minor but you can just return the result rather than storing in a variable of `workspace.spikesortingview` rather than create a whole variable for it. It doesn't matter here but for more memory intensive things it can matter.